test/statetest/statetest.hpp

```diff
@@ -68,6 +68,12 @@ struct StateTransitionTest
 template <typename T>
 T from_json(const json::json& j) = delete;
 
+template <>
+uint64_t from_json<uint64_t>(const json::json& j);
+
+template <>
+int64_t from_json<int64_t>(const json::json& j);
```

Comment:
I think we should remove `int64_t` in JSON. I expect we only use it for gas. So let's load it as `uint64_t`.

Reply:
Ok, this is bigger change, so let's keep it as is.